### PR TITLE
revisions for SLiM 4.2

### PR DESCRIPTION
**Pull request by bhaller**
Hi @jiseonmin and @petrelharp.  Here are revisions to `minimal.slim` for SLiM 4.2, plus a handful of other comments/changes.  Please propagate these changes to the other scripts (assuming you accept them), let me know that that has been done, and then I'll do a global review based on that.  I'm very pleased to have gotten these SLiM 4.2 changes in; the script is much more trim now!  (Especially getting rid of the need for rescheduleScriptBlock(), which I've always hated!)

### CODE REVIEW

File: minimal.slim [hunk 1]
```diff
@@ -5,30 +5,26 @@ initialize() {
 	// This model uses tree-sequence recording, but it is optional
 	initializeTreeSeq();
 	
+	// BCH: Hmm, putting the comments onto separate lines makes this twice as long.
+	// I'm definitely not a fan.  I liked them being on the same line as it was before;
+	// I just wanted spaces added before the comments so that they lined up, as I have
+	// done again below.  Does somebody dislike that style for some reason?  If so,
+	// then I guess I vote for the ragged comment line, as it was originally.  Taking
+	// up twice as much vertical space is my least favorite; and it's harder to figure
+	// out whether the comments refer to the line above or the line below, too.
```
Comment (petrelharp) [inline minimal.slim:14]: I like this (i.e., I like the comment-on-the-same-line style) also.

File: minimal.slim [hunk 2]
```diff
@@ -96,7 +89,7 @@ late() {
 	}
 }
 
-s1 late() {
+RUNTIME late() {	// BCH: rescheduleScriptBlock() no longer needed, yay!
```
Comment (petrelharp) [inline minimal.slim:92]: yesssss

Comment (jiseonmin) [reply minimal.slim:92]: Removed all reschedule blocks! It's very satisfying.

**Comment by jiseonmin** [comment]
I updated all of the scripts based on your comments and merged it. I made all comments to be on the same line in the parameter dictionary, but I am not sure if I am keeping them aligned. Is it just one tab between?

**Comment by bhaller** [comment]
So it sounds like I can close this PR and start a new review now, yes?

**Comment by bhaller** [comment]
OK, closing.  I'll start a new PR some time pretty soon, finishing up some other work...

**Pull request closed without merging**